src/components/GetStarted/Faq.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { ReactNode, useState, useCallback } from 'react'
+// @ts-ignore
+import { ButtonIcon, GU, RADIUS, useLayout, useTheme } from '@aragon/ui'
+import { Transition, animated } from 'react-spring/renderprops'
+import { fontWeight } from '../../style/font'
+import arrow from '../../assets/arrow.svg'
+
+const AnimatedDiv = animated.div
+
+type FaqProps = {
+  content: ReactNode
+  expansion: ReactNode
+}
+
+type ToggleButtonProps = {
+  onClick: () => void
+  opened: boolean
+}
+
+function Faq({ content, expansion }: FaqProps): JSX.Element {
+  const [opened, setOpened] = useState(false)
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const compactMode = layoutName === 'small'
+
+  const toggleButton = useCallback(() => {
+    setOpened((opened) => !opened)
+  }, [])
+
+  return (
+    <div
+      css={`
+        position: relative;
+        margin-bottom: ${RADIUS}px;
+      `}
+    >
+      <div
+        css={`
+          width: 100%;
+          z-index: 2;
+          height: auto;
+          padding: ${3 * GU}px ${8.5 * GU}px ${3 * GU}px ${3 * GU}px;
+          background: ${theme.surface};
+          box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.05);
+          border-radius: 8px;
```

Comment:
```suggestion
          border-radius: ${radius.medium};
```

Lets use a preset to keep it consistent with other elements